### DISCUSSION TITLE
How to add new fields to the template??

### DISCUSSION THREAD

**Discussion by sharukhkhan234** [horilla-opensource/horilla#220] [Q&A]
Hi!
I have added fields in the export Excel column, How can I add these fields in the employee template? for instance, if I want to download a template xls file, my added fields should display there in the imported xls file.

**Comment by asifrana1972** [comment]
How fields add or remove fieldwork_info_template.xlsx?

**Comment by horilla-opensource** [comment]
> Hi! I have added fields in the export Excel column, How can I add these fields in the employee template? for instance, if I want to download a template xls file, my added fields should display there in the imported xls file.

You have created new fields in the employee table and you want that to be imported from excel file or vice versa ?

**Reply by sharukhkhan234** [reply]
yes, I have created some fields and added them to the template, but when filling in that template, I can't import them.
